packages/auto-approve/src/auto-approve.ts

```diff
@@ -149,6 +149,18 @@ export function handler(app: Probot) {
       const repo = pr.pull_request.base.repo.name;
       const prNumber = pr.number;
 
+      // During codefreeze, simply set the RELEASE_FREEZE environment variable.
+      // if a PR is from release-please, it will not be merged:
+      if (
+        process.env.RELEASE_FREEZE &&
```

Comment:
I would have this explicitly check `=== "true"` otherwise, if set to "false" it would probably pass.

Reply:
I tend to prefer just treating a non empty string as `true`, since people might expect to be able to set `0`/`1` too, at which point yo're writing a bunch of logic for boolean parsing.

But happy to change if this is a blocker for you.

Reply:
```suggestion
        process.env.RELEASE_FREEZE === 'true' &&
```